composer.json

```diff
@@ -3,10 +3,11 @@
     "type": "symfony-bundle",
     "description": "The missing Symfony Admin Generator",
     "keywords": [
-        "Admin Generator",
         "admin",
+        "bundle",
```

Comment:
Already appears in the `type` field

Reply:
This is now used to update the github topics, since https://github.com/sonata-project/dev-kit/pull/1019

Moreover, I just look at the composer documentation
https://getcomposer.org/doc/04-schema.md#type

It recommends to only use custom type if we use custom logic for installation. Do we ?

Reply:
> This is now used to update the github topics, since sonata-project/dev-kit#1019

Maybe `bundle` could be obtained from some other source, like the presence of a class extending `Bundle`?

Reply:
Maybe we can get it from the `symfony-bundle` type.

But, Is it a bad idea to use the `bundle` or `symfony-bundle` keyword ? Someone can look for it on packagist.

Reply:
This package will already show up because it has bundle in its name, and `Symfony` in its description. So while it's not a bad idea, it's not needed, at least for search.

Reply:
Yes, but it won't show up when using the two most used keywords
![image](https://user-images.githubusercontent.com/9052536/93674784-0301f400-faac-11ea-9d76-aa72eb53f92d.png)

Reply:
Ah that's a good point indeed. Should definitely appear in https://packagist.org/?query=symfony&tags=symfony~admin